cmd/cmd.go

```diff
@@ -124,6 +163,7 @@ func parseFlags() {
 	flag.BoolVar(&verbose, "verbose", true, "If true, enable verbose logging")
 
 	flag.String("url", "http://localhost/", "URL to check")
+	flag.String("ws", "", "WS URL to check")
```

Comment:
Change to "wsURL" and "WebSocket URL to check"

Reply:
Okay, fxd